operations/observability/mixins/meta/rules/components/server/alerts.libsonnet

```diff
@@ -49,6 +49,120 @@
               description: 'Server cannot start workspace instances on workspace clusters.',
             },
           },
+          // Rollout alerts
+          {
+            alert: 'JsonRpcApiErrorRates',
+            // Reasoning: the values are taken from past data
+            expr: 'sum (rate(gitpod_server_api_calls_total{statusCode!~"2..|429"}[5m])) / sum(rate(gitpod_server_api_calls_total[5m])) > 0.04',
+            'for': '5m',
+            labels: {
+              // sent to the team internal channel until we fine tuned it
+              severity: 'warning',
+              team: 'webapp'
+            },
+            annotations: {
+              runbook_url: 'https://github.com/gitpod-io/runbooks/blob/main/runbooks/GitpodApiErrorRate.md',
+              summary: 'The error rate of the JSON RPC API is high. Investigation required.',
+              description: 'JSON RPC API error rate high',
+            },
+          },
+          {
+            alert: 'WebsocketConnectionRateHigh',
+            // Reasoning: the values are taken from past data
+            expr: 'sum(rate(server_websocket_connection_count[2m])) > 30',
+            'for': '5m',
+            labels: {
+              // sent to the team internal channel until we fine tuned it
+              severity: 'warning',
+              team: 'webapp'
+            },
+            annotations: {
+              runbook_url: 'https://github.com/gitpod-io/runbooks/blob/main/runbooks/WebsocketConnectionRateHigh.md',
+              summary: 'The websocket connection rate is higher than usual. Investigation required.',
+              description: 'Websocket connection rate high',
+            },
+          },
+          /**
+          * TODO(gpl) This will be true for US all the time. Can we exclude that cluster somehow?
+          * {
+          *   alert: 'db-sync not running',
+          *   expr: 'sum (kube_pod_status_phase{pod=~"db-sync.*"}) by (pod) < 1',
+          *   'for': '5m',
+          *   labels: {
+          *     // sent to the team internal channel until we fine tuned it
+          *     severity: 'warning',
+          *     team: 'webapp'
+          *   },
+          *   annotations: {
+          *     runbook_url: 'https://github.com/gitpod-io/runbooks/blob/main/runbooks/DbSyncNotRunning.md',
+          *     summary: 'The db-sync pod is not running. Investigation required.',
+          *     description: 'db-sync pod not running',
+          *   },
+          * },
+          *
+          */
+          {
+            alert: 'MessagebusNotRunning',
+            expr: 'up{job="messagebus"} < 1',
+            'for': '30s',
```

Comment:
That could be rather trigger happy.

Reply:
Whoops, that's a copy'n'paste issue indeed :laughing:

Reply:
Hm, on the other hand we want to get notified asap when this happesn. Let's make it 2mins, that should cover all "restart + image-pull" cases.